subset/brute/test_brute

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```

Comment:
let's put all this in subset/pentests as a grouping for other tests as well.  So -- subset/pentests/test_brute

Reply:
agreed changed to pentests